Packs/SimpleEmailReputation/Integrations/SimpleEmailReputation/SimpleEmailReputation.py

```diff
@@ -0,0 +1,141 @@
+import json
+import traceback
+from typing import Any, Dict, List, Optional, Tuple, Union, cast
+
+import dateparser
+import demistomock as demisto  # noqa: F401
+import urllib3
+from CommonServerPython import *  # noqa: F401
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+''' CLIENT CLASS '''
+
+
+class Client(BaseClient):
+    """Client class to interact with the service API
+
+    This Client implements API calls, and does not contain any Demisto logic.
+    Should only do requests and return data.
+    It inherits from BaseClient defined in CommonServer Python.
+    Most calls use _http_request() that handles proxy, SSL verification, etc.
+    """
+
+    def get_email_reputation(self, email: str) -> Dict[str, Any]:
+        """Gets the email reputation using the API endpoint
+
+        :type email: ``str``
+        :param email: email address to get the reputation for
+
+        :return: dict containing the email reputation as returned from the API
+        :rtype: ``Dict[str, Any]``
+        """
+
+        return self._http_request(
+            method='GET',
+            url_suffix='/' + email,
+        )
+
+
+''' COMMAND FUNCTIONS '''
+
+
+def email_reputation_command(client: Client, args: Dict[str, Any]) -> List[CommandResults]:
+    """email command: Returns email reputation for a list of emails
+
+    :type client: ``Client``
+    :param Client: client to use
+
+    :type args: ``Dict[str, Any]``
+    :param args:
+        all command arguments, usually passed from ``demisto.args()``.
+        ``args['email']`` is a list of emails or a single email
+
+    :return:
+        A ``CommandResults`` object that is then passed to ``return_results``,
+        that contains emails
+
+    :rtype: ``CommandResults``
+    """
+
+    emails = argToList(args.get('email'))
+    if len(emails) == 0:
+        raise ValueError('Email(s) not specified')
+
+    # Initialize an empty list of CommandResults to return
+    # each CommandResult will contain context standard for email
+    command_results: List[CommandResults] = []
+
+    for email in emails:
+        em_data = client.get_email_reputation(email)
+        em_data['email'] = email
+
+        readable_output = tableToMarkdown('Simple Email Reputation', em_data)
+
+        command_results.append(CommandResults(
+            readable_output=readable_output,
+            outputs_prefix='Emailrep.email',
+            outputs_key_field=em_data['email'],
+            outputs=em_data
+        ))
+    return command_results
+
+
+def test_module(client: Client):
+    em_data = client.get_email_reputation('test@test.com')
+    return em_data
```

Comment:
You return `em_data` but never use it, so it might be a good practice to return "ok" after calling to `client.get_email_reputation('test@test.com')`